packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Basic.Example.tsx

```diff
@@ -1,80 +1,60 @@
+import { AnimationClassNames, mergeStyleSets, getTheme } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
 import { Layer } from 'office-ui-fabric-react/lib/Layer';
 import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
 
-interface ILayerBasicExampleContext {
-  message?: string;
-}
-
-const LayerBasicExampleContext = React.createContext<ILayerBasicExampleContext>({ message: undefined });
-
-interface ILayerContentExampleState {
-  time: string;
-}
-
-class LayerContentExample extends BaseComponent<{}, ILayerContentExampleState> {
-  public state = {
-    time: new Date().toLocaleTimeString()
-  };
-
-  public componentDidMount() {
-    this._async.setInterval(() => {
-      this.setState({
-        time: new Date().toLocaleTimeString()
-      });
-    }, 1000);
-  }
-
-  public render() {
-    return (
-      <LayerBasicExampleContext.Consumer>
-        {value => (
-          <div className={css(styles.content, AnimationClassNames.scaleUpIn100)}>
-            <div className={styles.textContent}>{value.message}</div>
-            <div>{this.state.time}</div>
-          </div>
-        )}
-      </LayerBasicExampleContext.Consumer>
-    );
+// tslint:disable:jsx-no-lambda
+
+const theme = getTheme();
+const classNames = mergeStyleSets({
+  content: {
+    backgroundColor: theme.palette.themePrimary,
+    color: theme.palette.white,
+    lineHeight: '50px',
+    padding: '0 20px',
+    display: 'flex'
+  },
+  textContent: {
+    flexGrow: '1'
   }
-}
-
-interface ILayerBasicExampleState {
-  showLayer: boolean;
-}
-
-export class LayerBasicExample extends BaseComponent<{}, ILayerBasicExampleState> {
-  public state = {
-    showLayer: false
-  };
-
-  public render() {
-    const { showLayer } = this.state;
-    return (
-      <LayerBasicExampleContext.Provider
-        value={{
-          message: 'Hello world.'
-        }}
-      >
-        <div>
-          <Toggle label="Wrap the content box below in a Layer" inlineLabel checked={showLayer} onChange={this._onChange} />
-
-          {showLayer ? (
-            <Layer>
-              <LayerContentExample />
-            </Layer>
-          ) : (
-            <LayerContentExample />
-          )}
-        </div>
-      </LayerBasicExampleContext.Provider>
-    );
-  }
-
-  private _onChange = (ev: React.FormEvent<HTMLElement | HTMLInputElement>, checked: boolean): void => {
-    this.setState({ showLayer: checked });
-  };
-}
+});
+
+const LayerContentExample: React.FunctionComponent = () => {
+  const [time, setTime] = React.useState<string>(new Date().toLocaleTimeString());
+
+  React.useEffect(() => {
+    const interval = setInterval(() => setTime(new Date().toLocaleTimeString()));
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <div className={css(classNames.content, AnimationClassNames.scaleUpIn100)}>
+      <div className={classNames.textContent}>Hello world</div>
+      <div>{time}</div>
```

Comment:
wouldn't this print the time in the snapshot tests for examples?

Reply:
Also, I think in general we should avoid unnecessary code in examples and make them as concise as possible.

Reply:
The snapshot tests mock Date to be a constant. But come to think of it I have no idea why this example includes a clock, so I can remove that part.